xapi-xenopsd-xc.opam

```diff
@@ -1,11 +1,35 @@
 opam-version: "1.2"
-name: xapi-xenopsd-xc
+name: "xapi-xenopsd-xc"
 maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://github.com/xapi-project/xenopsd"
+dev-repo: "https://github.com/xapi-project/xenopsd.git"
+bug-reports: "https://github.com/xapi-project/xenopsd/issues"
 build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
-build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 depends: [
   "jbuilder" {build}
   "xapi-xenopsd"
+  "bisect_ppx"
+  "cohttp"
+  "core"
+  "ezxenstore"
+  "fd-send-recv"
+  "oclock"
+  "qmp"
+  "re"
+  "rpc"
+  "sexplib"
+  "uri"
+  "uuidm"
+  "uutf"
+  "xapi-forkexecd"
+  "xapi-idl"
+  "xapi-inventory"
+  "xapi-stdext-unix"
+  "xenctrl"
+  "xenstore"
+  "xenstore_transport"
+  "xmlm"
+  "xcp"
+  "odoc" {doc}
```

Comment:
Now that we're declaring `{doc}` dependencies, we should also add instructions about how to build the docs using the `build-doc` field (documented here: http://opam.ocaml.org/doc/Manual.html) - this applies to all opam files with a `{doc}` dependency.